### PR TITLE
Django3 migration

### PR DESCRIPTION
**Pull request by robincherbonnier**
- Added support for Django versions 3 and 4
- Added Github actions CI

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: '2.1'
+services:
+
+  postgres:
+    image: postgres:10
```
Comment (alyohea) [inline docker-compose.yml:5]:
```suggestion
    image: postgres:12
```

lets use postgres12

Comment (robincherbonnier) [reply docker-compose.yml]: Done.

File: requirements_test.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
 coverage==4.4
-psycopg2>=2.6.2
+psycopg2>=2.8,<2.9
```
[the remark this answers is not in the document]
Comment (robincherbonnier) [reply requirements_test.txt]: Thanks. Done.

**Review by alyohea — APPROVED** [review]
👍

**Comment by aleksandr-shtaub** [comment]
🎈
Can you also add changelog to the [HISTORY.rst](../blob/master/HISTORY.rst)?

**Comment by robincherbonnier** [comment]
Thank you @alyohea and @aleksandr-shtaub for your reviews.
I changed HISTORY.rst.
If it's OK for you I'll merge it and make a PR for the release.

**Pull request merged**